### PR TITLE
README add ENV variable setup

### PR DESCRIPTION
**Pull request by notapatch**

### CODE REVIEW

**Comment by EGiataganas** [comment]
This is a great addition. Although I think these steps should be in the terraform repo

**Comment by notapatch** [comment]
My initial thought was to put it into Terraform's README as well.

File: README.md [hunk 1]
```diff
@@ -165,3 +165,27 @@ After adding a new Stimulus controller run `./bin/rails stimulus:manifest:update
 ## Front end components
 
 As much as possible, we follow the GOV.UK Design System. You will find most of the HTML components you need [here](https://design-system.service.gov.uk/get-started). For help with forms we use the [GOV.UK Ruby on Rails Form Builder gem](https://govuk-form-builder.netlify.app). See [here](https://github.com/unboxed/bops/blob/main/app/views/users/_form.html.erb) for a simple example of implementation.
+
+## Environmental variables
+
+Environmental variables are defined in the bops-terraform project using Ansible.
```
Comment (allbecauseyoutoldmeso) [inline README.md:171]: Would it be worth adding a (github) link to the file where they're defined?

File: README.md [hunk 2]
```diff
@@ -165,3 +165,27 @@ After adding a new Stimulus controller run `./bin/rails stimulus:manifest:update
...
+set as environmental variables.
+
+```python
+PAAPI_URL={{ lookup('aws_ssm', '/bops/' + ansible_env + '/PAAPI_URL', region=aws_region) }}
+PLANNING_HISTORY_ENABLED={{ lookup('aws_ssm', '/bops/' + ansible_env + '/PLANNING_HISTORY_ENABLED', region=aws_region) }}
+```
+
+In this case ansible_env can typically be preview or production
+
+## AWS Parameter store is a key value pair store
```
Comment (allbecauseyoutoldmeso) [inline README.md:182]: Could we also have a link to the relevant page in the AWS console? 🤔

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Very happy to see this!

**Comment by allbecauseyoutoldmeso** [comment]
> My initial thought was to put it into Terraform's README as well -[slack discussion was here](https://ubxd.slack.com/archives/C010XKV8QLR/p1670950859692319?thread_ts=1670949377.776049&cid=C010XKV8QLR).
> 
> @benbaumann95 Argument was:
> 
> I think it’s more intuitive to see that detailed out on the [main repo](https://github.com/unboxed/bops) rather than [terraform](https://github.com/unboxed/bops-terraform) as that’s probably not where i would have looked initially trying to do this.
> 
> Obviously, I'm happy with either option.

I think it would be fine to have it in both, but definitely good to have here as I'm not sure everyone will know to look at the terraform repo when they're still getting used to the set up!

**Comment by notapatch** [comment]
What we need is another section to introduce the terraform project it's not the job of the ENV variable section to introduce terraform project. 1 step at time..... First ENV.

**Review by allbecauseyoutoldmeso — APPROVED** [review]
🥇

**Comment by allbecauseyoutoldmeso** [comment]
> What we need is another section to introduce the terraform project it's not the job of the ENV variable section to introduce terraform project. 1 step at time..... First get ENV section in.

Agreed and agreed!

**Pull request merged**